Single-Sign-on/faqs.md

```diff
@@ -0,0 +1,165 @@
+
+For step by step instructions on how to configure Docker SSO, see [SSO docs](/single-sign-on).
+
+### Q: Is it possible to use more than one IdP with Docker SSO?
+
+No. You can only configure Docker SSO to work with a single IdP. A domain can only be associated with a single IdP.Docker currently supports Azure AD and identity providers that support SAML 2.0
+
+### Q: Is it possible to change my identity provider after configuring SSO?
+
+Yes. You must delete your existing IdP configuration in Docker Hub and follow the instructions in the [SSO docs](/single-sign-on) to configure SSO using your IdP.
+
+###  Q: Is Docker SSO available for all paid subscriptions?
+
+Docker SSO is only available with the Docker Business subscription. [Upgrade](/subscription/upgrade/){:target="blank" rel="noopener" class=""} your existing subscription to start using Docker SSO.
+
+### Q: Does Docker SSO support multi-factor authentication (MFA)?
+
+When SSO is being used by an organization, MFA is determined at the idP level and not the Docker system.
+
+### Q: How does service accounts work with SSO?
+
+Service accounts work like any other user when SSO is turned on. If the service account is using an email for a domain with SSO turned on, it needs a [PAT](/docker-hub/access-tokens/) for CLI and API usage.
+
+
+## Configuration
+
+### Q: What information do I need from my Identity providers to configure SSO?
+
+To enable SSO in Docker, you need the following from your IdP:
+
+* **SAML 2.0**: Entity ID, ACS URL, Single Logout URL and Certificate Download URL
+* **Azure AD**: Client ID, Client Secret, AD Domain
+
+### Q: Is DNS verification required to enable SSO?
+
+Yes. You must verify a domain before using it with an SSO connection.
+
+### Q: Does Docker SSO support authenticating through the command line?
+
+Yes. When SSO is enabled, you can access the Docker CLI through Personal Access Tokens (PATs).  Each user must create a PAT to access the CLI. To learn how to create a PAT, see [Managing access tokens](/docker-hub/access-tokens/).
+
+###  Q:  How does SSO affect our automation systems and CI/CD pipelines?
+
+Before enforcing SSO, you must create PATs for automation systems and CI/CD pipelines and use the tokens instead of a password.
+
+When SSO is enforced, password-based authentication no longer works on your automation systems and CI/CD pipelines.
+
+### Q: Do I need a specific version of Docker Desktop for SSO?
+
+Yes, all users in your organization must upgrade to Docker Desktop version 4.4.0 or higher. Users on older versions of Docker Desktop will not be able to sign in after enforcing SSO.
+
+### Q: Does SAML authentication require additional attributes?
+
+You must provide an email address as an attribute to authenticate via SAML. The ‘Name’ attribute is currently optional.
+
+### Q: Does the application recognize the NameID/Unique Identifier in the SAMLResponse Subject? If so please provide the format you require.
+
+TBD
```

Comment:
Could you remove this if we don't have any info yet?

Reply:
Deleting it for now @usha-mandya